package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -503,6 +512,11 @@ def __init__(self, filename, lammps_coordinate_convention="auto",
                              " is not a valid option. "
                              f"Please choose one of {option_string}")
 
+        if unwrap_images:
+            self._unwrap = unwrap_images
+        else:
+            self._unwrap = False
```

Comment:
```suggestion
        self._unwrap = unwrap_images
```
If the default is `False` then the `if` / `else` shouldn't be necessary?